### ISSUE TITLE
Google Plus Linked Account

### ISSUE THREAD

**Issue by elenatorro** [CartoDB/cartodb#13113]
### Context

We are currently using an `iframe` to unlink the user's Google account from CARTO. In the FE/BE Split, we can not use iframe and there is no way to unlink a user's Google account directly, so the best option in my opinion is to redirect the user to this [link](https://myaccount.google.com/permissions?pli=1) so she can unlink the account from there.

However, we do no have any value that indicates us if the account is linked or not, so we have to see if we can send this from backend through the `/me` endpoint (under demand, by using a parameter).

Actually, I think there is a little bug: when the user unlinks an account, and is not able to see the option to unlink, the account form is still showing the `Your account is linked to your Google account` message. 

<img width="595" alt="captura de pantalla 2017-11-22 a las 13 09 11" src="https://user-images.githubusercontent.com/3824953/33127559-34c16846-cf89-11e7-8e63-206446e006b1.png">
 
I think this we should not invest time now to fix that bug, because this page will be served in the client soon.

So, as I said, I consider the best solution is to be able to request through the API if the user's account is rather linked or not in order to show the link and the `Your account is linked to your Google account` message. Another option would be to delete that button, since it is not mandatory to show a way to unlink a Google account. I think it is better to show it, but I would like to know your opinion.

@CartoDB/backend @nobuti @xavijam

**Comment by juanignaciosl** [comment]
I’d go for `/me?check_google_login_enabled=true`: it doesn’t harm `/me` performance and it's an API-centric solution.

If it's blocking, as we're currently swamped, I'd suggest to unblock this by _not_ adding the button (it's not mandatory), move on, and add the button once `/me` supports the check.

**Comment by juanignaciosl** [comment]
@elenatorro I'm trying to prioritize issues. Did you move on not adding the disconnect button? I'd like to know whether we must do a proper fix for this or if it can be postponed.
